src/daemon/jsonrpc/api.rs

```diff
@@ -91,6 +91,10 @@ pub trait RpcApi {
     #[rpc(meta, name = "getinfo")]
     fn getinfo(&self, meta: Self::Metadata) -> jsonrpc_core::Result<serde_json::Value>;
 
+    // Print all available commands
```

Comment:
nit: This should be a [doc comment](https://doc.rust-lang.org/reference/comments.html) to match all other methods